PULL_REQUEST_MANAGEMENT.md

```diff
@@ -0,0 +1,14 @@
+# Rules for managing Pull Requests
+
+For the sake of trying to maintain an acceptable number of open but idle PRs, the following rules should be considered:
+- If a PR has at least one approval, it can be merged after 1 week of waiting for additional comments.
+  - If you consider the change trivial enough or it has at least 3 approvals, it may also be merged right away.
+- If a PR stays in "changes requested" for 2 weeks, and that change has not been done in the meantime, it shall be closed.
+- If you require a review from a particular person, assign the PR to that person. Don't just rely on the "review requested" feature of GitHub.
+- Don't feel obliged to comment everything you see, just for the sake of commenting. Be it on JIRA, GitHub, or even on IRC.
+
+In addition, in order to avoid coming off as rude to helpful contributors, please refrain from
```

Comment:
Still missing a colon :stuck_out_tongue:

Reply:
My intention with that was to format it as a split sentence, like:

>In addition, in order to avoid coming off as rude to helpful contributors, please refrain from
> - Asking the contributor to do unrelated work,
> - Editing the code without asking first,
> - Closing without providing a reason, or
> - Merging with the intention to rewrite that code soon after.

But I didn't do that so I guess I could also add that colon :P